slasher/main.go

```diff
@@ -95,6 +95,15 @@ func main() {
 			}
 		}
 
+		if !ctx.Bool(featureconfig.AltonaTestnet.Name) &&
+			!ctx.Bool(featureconfig.OnyxTestnet.Name) &&
+			!ctx.Bool(featureconfig.MedallaTestnet.Name) &&
+			!ctx.Bool(featureconfig.SpadinaTestnet.Name) &&
+			!ctx.Bool(featureconfig.ZinkenTestnet.Name) {
+				log.Error("testnet is not specified, required: --<testnet-name>")
+				os.Exit(1)
+		}
```

Comment:
This code is identical to the code in beacon-chain/main.go. Can you abstract this into a shared method of some kind?

Also, please return an error instead of os.Exit.